spec/i18n_routable/i18n_routable_spec.rb

```diff
@@ -149,55 +153,55 @@ def resolve(path, http_verb=:get)
 
     it 'should understand the difference between display locales and backend locales' do
       I18n.locale = :gibb
-      new_post_url.should eql "http://www.example.com/gibberish/posts/the-new"
+      new_post_url.should eql "http://www.example.com/gibb/posts/the-new"
```

Comment:
Monorail might still have some usage of display_locale. Or more likely there is still code that uses it, but it's not a used feature. So we might be able to remove it if so.